product_quick_bom/models/product.py

```diff
@@ -0,0 +1,34 @@
+#   Copyright (C) 2015 Akretion (http://www.akretion.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    @api.depends("bom_ids", "bom_ids.active")
+    def _compute_bom_id(self):
+        for rec in self:
+            if len(rec.bom_ids.ids) == 1:
+                rec.bom_id = rec.bom_ids[0]
+            else:
+                rec.bom_id = self.env["mrp.bom"]
```

Comment:
```suggestion
            rec.bom_id = rec.bom_ids[0] if len(rec.bom_ids.ids) == 1 else self.env["mrp.bom"]
```

Reply:
Hi Mr @DucTruongKomit 
It seems `.ids` is redundant, how about only `rec.bom_ids`?